### DISCUSSION TITLE
Kimber Soiling Model default rainfall thresholds and grace periods

### DISCUSSION THREAD

**Discussion by ShaLi2021E** [pvlib/pvlib-python#1579] [Q&A]
Hello everyone,

Has anyone/any research performed study on setting the appropriate soiling loss rate/rainfall thresholds/grace periods in other climate region and soiling environment type other than Central Valley/Northern & Southern CA and Desert climate which are originally referenced in Kimber Model 2006? 

Thank you!
Sha

**Comment by mikofski** [comment]
In my experience, only the build up rate changes. I think Zoe Defreitas and Mike Deceglie have published a few on this:
* https://ieeexplore.ieee.org/document/7749992
* https://ieeexplore.ieee.org/document/8981194

**Comment by matt14muller** [comment]
Look at the NREL soiling map https://www.nrel.gov/pv/soiling.html for some data that has been collected on soiling rates.  We have some additional papers behond what was mentioned above (search Michael Deceglie, Matthew Muller, Leonardo Micheli as we all have various papers on this).  I use 0.75 mm/day as a threshold below which I assume no cleaning although the 1000s of sites I have looked at have shown no consistent value for minimum rainfall.  I assume no grace period although I am not claiming that this is always true.
